setup/install/halyard_migration.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
I think you'll need to either add a sentence in halyard.md that refers to this doc or add this to _data/navigation.yml. Or  both? Unless you only want this page accessible by direct link, like if someone asks us a question about migrating and we send them the link.

Reply:
Lars was of the mindset of people finding out about this page only when they actually needed to migrate. So I added a check in the install script that basically says, if you already have halyard installed from the debian package, give the user the link to this page.

You can see that change here: https://github.com/spinnaker/halyard/pull/806

Reply:
Ah yes, I remember now. Perf. Approved.

Reply:
"Perf" here meaning "perfect." ;)

Reply:
;) We can always add that link you are suggesting later if it actually be comes a thing we get asked about though. Thanks for the review! This ended up much better IMO than when I first sent it.